benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/HubConnectionContextBenchmark.cs

```diff
@@ -30,11 +30,16 @@ public class HubConnectionContextBenchmark
         [GlobalSetup]
         public void GlobalSetup()
         {
-            using (var memoryBufferWriter = new MemoryBufferWriter())
+            var memoryBufferWriter = MemoryBufferWriter.Get();
```

Comment:
Can still use `using` and `Dispose` rather than `try`/`finally`?

Reply:
Yea, I think we can consider doing this for everything but after.